packages/website/docusaurus.config.js

```diff
@@ -1,193 +1,10 @@
-// @ts-check
-// Note: type annotations allow type checking and IDEs autocompletion
+'use strict';
 
-const remarkPlugins = [
-  [require('@docusaurus/remark-plugin-npm2yarn'), { sync: true }],
-];
+require('ts-node').register({
+  scope: true,
+  scopeDir: __dirname,
+  swc: true,
+  transpileOnly: true,
+});
```

Comment:
This is art 👏🏻

We will support `docusaurus.config.ts` some time in the near future, because we'll probably need the same infrastructure to support ESM config files